src/miniscript/context.rs

```diff
@@ -332,7 +339,7 @@ where
     /// Note that this includes the serialization prefix. Returns
     /// 34/66 for Bare/Legacy based on key compressedness
     /// 34 for Segwitv0, 33 for Tap
-    fn pk_len<Pk: MiniscriptKey>(pk: &Pk) -> usize;
+    fn pk_len<Pk: MiniscriptKey>(pk: &KeyExpr<Pk>) -> usize;
```

Comment:
As mentioned earlier, we should not change this.

Reply:
But, we should add another method here with default implementation as 

```rust
fn key_expr_len<Pk: MiniscriptKey>(pk: &KeyExpr) {
	match pk {
		SingleKey(pk) => Ctx::pk_len(&pk),
		Musig(expr) => 33, // not 32 as mentioned in another comment below
	}
}
```

Reply:
Ok, created the new method with the default implementation.